composer.json

```diff
@@ -30,6 +30,10 @@
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         },
+        {
+            "type": "git",
+            "url": "https://github.com/openeuropa/rdf_skos.git"
+        },
```

Comment:
Why was this required?

Reply:
Hmm, whenever I used the Dev branch, I had to add the repository because otherwise, it wouldn’t work. But here, it actually works without it. 
OK, now I know. If its private repo you have to add.
OK, I’ll remove it.